### ISSUE TITLE
Automatic input data existence check should disconsider the phedex group

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#9484]
**Impact of the new feature**
ReqMgr2MS

**Is your feature request related to a problem? Please describe.**
While checking my private tests on the MicroServices, I noticed that some blocks were at a given disk location, but it wasn't seen by the service. It turns out we match the `blockreplicas` output against a given PhEDEx group (in this case `DataOps`).

**Describe the solution you'd like**
After chatting with Sharad, he confirmed that Unified doesn't care whether the dataset/block is subscribed under a PhEDEx group different than `DataOps`. If data is available, this is all that matters to proceed with the workflow assignment/execution. 

In short, we should remove the phedex group check when evaluating which replicas are already in place. If data is complete and subscribe, skip it from the data transfer.

**Describe alternatives you've considered**
Which is different than the data placement. For that, we do have to define the `DataOps` group, always.

**Additional context**
Problem spotted with this DMWM/TaskChain_NanoAOD template.
